frontend/website/pages/docs/developers/sandbox-container.mdx

```diff
@@ -0,0 +1,47 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Sandbox Container"});
+
+# Sandbox Container
+
+If all you need is a running daemon and some blocks, the sandbox container has everything you need! It uses the same configuration as the testnet, but instead of the community participants ledger it uses a simple ledger with a single demo account.
+
+This can be useful for testing or playing with some of the core features of the protocol.  This node supports multiple accounts, sending transactions between them, and will also support delegating and staking. In fact since it's a single node network, you'll get all the block rewards!
```

Comment:
My experience with delegating stake is this doesn't work. Has this been resolved (I've not specifically tried with this build but see  https://discordapp.com/channels/484437221055922177/601215883939020852/699288425257500672